### DISCUSSION TITLE
Unzip a splitted ZIP

### DISCUSSION THREAD

**Discussion by pedro-tugame** [aonez/Keka#1117] [Show and tell]
Hi,

I've searched for a discussion on the same topic, and I have not found it, so I hope I'm not asking something that was addressed before.

I've downloaded KEKA because when I download folders from google drive to my MAC it creates a split zip file of around 2GB each.

When I try to unzip it, it doesn't create a single file (folder) maintaining the original folder structure.

Could you please let me know how I can do that with KEKA?

Thanks for your help :)

**Comment by aonez** [comment]
I remember talking about this sometime. Are those ZIP files ending with an extension like zip.001 ... zip.005?

If not (most probably) they're not conforming a split volume, they're rather separated zip files. If that is the case your best option is to merge the extracted folders, copy the contents inside one folder and choose "merge" when Finder asks.

**Reply by pedro-tugame** [reply]
Hi Aonez,

Thanks for your msg :)

When I download a folder from google drive (with multiple folders and files inside) it splits it into multiple ZIP files.

The extension of those zip files are .ZIP
They indeed have 001, 002, etc, buts it's before .ZIP not afterward.

FolderName-001.zip
FolderName-002.zip
FolderName-003.zip
etc

**Reply by aonez** [reply]
So do the merge thing, they're not a split volume, sadly.

**Reply by pedro-tugame** [reply]
Hi Aonez,
Could you please explain how I "merge" them...

I've tried to unzip all the folders and afterward selecting them and also moving then into a new folder, but I have not seen an option of "merge"

Thanks for your help :)
